### PR TITLE
Upgrade to v5 of react-select.

### PR DESCRIPTION
**Pull request by Andrekra**
Fixes issue https://github.com/jacobworrel/react-windowed-select/issues/82

### CODE REVIEW

File: src/MenuList.tsx [hunk 1]
```diff
@@ -17,14 +16,18 @@ interface ListChildProps extends ListChildComponentProps {
   style: Style
 }
 
+interface OptionTypeBase {
```
Comment (Andrekra) [inline src/MenuList.tsx:19]: OptionTypeBase is removed from react-select. According to the upgrade docs:

> Options can be any type (if using getOptionValue and getOptionLabel) so there's no longer a base type for options

They've mapped it to unknown, but I pulled the old type back that's basically any object.

File: src/util.ts [hunk 2]
```diff
@@ -81,5 +81,3 @@ export function createGetHeight ({
     }
   };
 }
-
-export const sum = (a: number, b: number): number => a + b;
```
Comment (Andrekra) [inline src/util.ts:85]: It was not used, just some dead code.

**Review by Andrekra — COMMENTED** [review]
I was not sure which npm version was used. So the lock file got an update after install

`npm WARN old lockfile
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile
`

**Comment by jacobworrel** [comment]
Thanks @Andrekra ! Sorry I haven't gotten around to this but I will try to take a look soon.

**Review by jacobworrel — APPROVED** [review]
LGTM - I'll bump the package version to 5.0.0 and then publish.

**Pull request merged**